packages/accounts-passwordless/passwordless_server.js

```diff
@@ -93,13 +93,13 @@ Accounts.registerLoginHandler('passwordless', options => {
 
 // Utility for plucking addresses from emails
 const pluckAddresses = (emails = []) => emails.map(email => email.address);
-const createUser = userData => {
+const createUser = async userData => {
   const { username, email } = userData;
   if (!username && !email) {
     throw new Meteor.Error(400, 'Need to set a username or email');
   }
   const user = { services: {} };
-  return Accounts._createUserCheckingDuplicates({
+  return await Accounts._createUserCheckingDuplicates({
```

Comment:
I believe it's a matter of taste, but usually, using return await isn't necessary. You can typically return the promise directly in an async function.

Reply:
I know it might seem redundant but it's just clear.